psi4/driver/driver.py

```diff
@@ -1269,13 +1275,8 @@ def hessian(name, **kwargs):
         core.print_out(
             """hessian() will perform frequency computation by finite difference of analytic gradients.\n""")
 
-        # Shifting the geometry so need to copy the active molecule
-        moleculeclone = molecule.clone()
-
         # Obtain list of displacements
-        findif_meta_dict = driver_findif.hessian_from_gradient_geometries(moleculeclone, irrep)
-        moleculeclone.reinterpret_coordentry(False)
-        moleculeclone.fix_orientation(True)
+        findif_meta_dict = driver_findif.hessian_from_gradient_geometries(molecule, irrep)
```

Comment:
Should this be `clone` here?

Reply:
No.

None of the geometry generators change the molecule, so there's no need to have a clone for purposes of generating the metadict. Cloning is only needed for the computations and is now done in `_process_displacement`, because different displacements need different clones for symmetry purposes.